### DISCUSSION TITLE
Meta-badges / Quests / Multi-game sets

### DISCUSSION THREAD

**Discussion by oddgoo** [RetroAchievements/RAWeb#1415] [Ideas]
This represents the ability to have a new type of site badge/award that is made up from a variety of simple conditions and achievements from multiple games.
Examples:

> **So long-a Bowser!**
> *Defeat Bowser in all 5 main-line Super Mario Games*
> 
> **World Tour**
> *Master 1 game in all Retroachievement Systems*
> 
> **Nintendo Hard**
> *Master 45 games for the Nintendo Entertainment System*
> 
> **Sore Thumbs**
> *Get the Street Fighter Achievement and Master 10 other games from the Fighting Collection*

There is a related discussion, tho less elaborate and only considers single achievements: https://github.com/RetroAchievements/RAWeb/discussions/667

This is inspired by the manually managed competitions and other sites like TrueAchievements, but these would be permanent and automatic.

## Requirements / Admin

Composing this from an Admin / DEV point of view would be represented by a simple listing of "AND" requirements.
I can see 4 fields for each requirements. Example:

| Requirement Type | Identifier  | Value   | Condition (optional)   | 
| ---------------- | ----------- | ----------- | ----------- |
| Single Achievement                 | Achievement ID (ex: 8896)      | N/A     | N/A     |
| Any Achievements from HUB          | Collection/HUB Id (eg: 8835)    | Number of Achievements to Obtain (eg: 10)       | N/A       |
| Any Achievements from System       | System ID (eg: 6 )     | Number of Achievements to Obtain (eg: 50)     | No Hacks/Homebrew     |

A Meta-badge / Quest would need just 1 requirement minimum, but extra ones could be added and removed.
The badge is of course awarded when all requirements are met.

**Requirement Type:** Enum / Dropdown
**Identifier:** Text Field
**Value:** Enum / Dropdown
**Condition 2:** Enum / Dropdown

## Extra thoughts

Given the Nature of constantly-changing collections, etc. Awards would be awarded on first completion, rather than constantly re-checked (same as current game bades).

For assets, these could give the same type of square awards as events/games, but we can also have a bigger rectangular banner to have a fancier / more distinguishing listing page. Quick example:

![image](https://user-images.githubusercontent.com/4585118/222990675-0a40d8d6-d028-43fe-b602-55d5dc5ea8ec.png)
![image](https://user-images.githubusercontent.com/4585118/222990764-d3f216de-3bde-475d-8b0a-34d278809a2a.png)

**Comment by Jamiras** [comment]
related: #667

**Comment by wescopeland** [comment]
Superseded by ~~https://github.com/RetroAchievements/RAWeb/discussions/1415~~ https://github.com/RetroAchievements/RAWeb/discussions/4194.

**Reply by oddgoo** [reply]
Awesome! (I think you meant #4194)

**Reply by wescopeland** [reply]
You're right - I did 😅. Edited, thank you!
